lineapy/plugins/jinja_templates/dockerfile.jinja

```diff
@@ -1,4 +1,4 @@
-FROM apache/airflow:latest
+FROM apache/airflow:latest-python{{ python_version }}
```

Comment:
not sure why this change is showing up here.. this should already be merged into main.

Reply:
This is perhaps due to my omission to rebase onto the latest version of `main`